services/hanke-service/src/main/resources/email/template/taydennyspyynto-peruttu.subject.mustache

```diff
@@ -0,0 +1 @@
+Haitaton: Hakemustasi ({{hakemustunnus}}) koskeva täydennyspyyntö on peruttu / Hakemustasi ({{hakemustunnus}}) koskeva täydennyspyyntö on peruttu / Hakemustasi ({{hakemustunnus}}) koskeva täydennyspyyntö on peruttu
```

Comment:
This is as it is described in the specs but I noticed that the email for information request does not have `hakemustunnus` in its subject.

Reply:
I think the subject should be as short as possible since it's repeated three times for different languages. I removed the hakemustunnus from the subject line.

Reply:
Okay. But maybe then the specs should be updated as well?

Reply:
The specs have been updated and they match the current implementation.